third_party/openmetadata/Makefile

```diff
@@ -6,8 +6,20 @@ deploy-openmetadata:
 	helm repo add open-metadata https://helm.open-metadata.org/ || true
 	helm repo update || true
 	kubectl create ns ${INSTALLATION_NAMESPACE} || true
+ifeq ($(IBM_OPENSHIFT_INSTALLATION), true)
+	oc adm policy add-scc-to-user anyuid -z airflow -n ${INSTALLATION_NAMESPACE}
+	oc adm policy add-scc-to-user anyuid -z mysql -n ${INSTALLATION_NAMESPACE}
+	oc adm policy add-scc-to-user ibm-privileged-scc -z default -n ${INSTALLATION_NAMESPACE}
+	kubectl apply -f ibm-openshift/pvc1.yaml -n ${INSTALLATION_NAMESPACE} || true
+	kubectl apply -f ibm-openshift/pvc2.yaml -n ${INSTALLATION_NAMESPACE} || true
+	kubectl apply -f ibm-openshift/pvc3.yaml -n ${INSTALLATION_NAMESPACE} || true
+	kubectl apply -f ibm-openshift/pvc4.yaml -n ${INSTALLATION_NAMESPACE} || true
+else
 	sed s/INSTALLATION_NAMESPACE/${INSTALLATION_NAMESPACE}/g pv1.yaml | kubectl apply -n ${INSTALLATION_NAMESPACE} -f - || true
+	kubectl apply -f pvc1.yaml -n ${INSTALLATION_NAMESPACE} || true
 	sed s/INSTALLATION_NAMESPACE/${INSTALLATION_NAMESPACE}/g pv2.yaml | kubectl apply -n ${INSTALLATION_NAMESPACE} -f - || true
+	kubectl apply -f pvc2.yaml -n ${INSTALLATION_NAMESPACE} || true
+endif
 	kubectl create secret generic airflow-mysql-secrets --from-literal=airflow-mysql-password=${AIRFLOW_MYSQL_PASSWORD} -n ${INSTALLATION_NAMESPACE} || true
 	helm install openmetadata-dependencies open-metadata/openmetadata-dependencies --version ${OPENMETADATA_HELM_CHART_VERSION} --values values-deps.yaml --wait --timeout 20m -n ${INSTALLATION_NAMESPACE} \
```

Comment:
can you add `@` before helm install to prevent its printing? Otherwise it prints username and password settings
```
--set airflow.airflow.users[0].username=admin \
--set airflow.airflow.users[0].password=admin \
--set airflow.airflow.users[0].firstName=Peter \
--set airflow.airflow.users[0].lastName=Parker \
--set airflow.airflow.users[0].email=spiderman@superhero.org \
--set airflow.externalDatabase.user=airflow_user \
--set airflow.airflow.config.AIRFLOW__LINEAGE__OPENMETADATA_API_ENDPOINT="http://openmetadata.open-metadata.svc.cluster.local:8585/api" \
--set airflow.externalDatabase.host=mysql.open-metadata.svc.cluster.local \
--set mysql.auth.rootPassword=password || true
```